Documentation/nvme-connect-all.txt

```diff
@@ -38,6 +38,7 @@ SYNOPSIS
                 [--nbft]
                 [--no-nbft]
                 [--nbft-path=<STR>]
+                [--application=<id>]
```

Comment:
I'm wondering if we should define this as `--application=`**`<STR>`** to make it clear that a string is expected.

Reply:
yes, that make sense.

I am wondering if we should call it `--context` or something similar. `--application` seems a bit wierd. e.g.

```systemd
[Unit]
Description=Connect NVMe-oF subsystems automatically during boot
ConditionPathExists=|@SYSCONFDIR@/nvme/config.json
ConditionPathExists=|@SYSCONFDIR@/nvme/discovery.conf
After=network-online.target
Before=remote-fs-pre.target

[Service]
Type=oneshot
ExecStartPre=/sbin/modprobe nvme-fabrics
ExecStart=@SBINDIR@/nvme connect-all --application=autoboot

[Install]
WantedBy=default.target
```

Reply:
>
> I am wondering if we should call it `--context` or something similar. `--application` seems a bit wierd. e.g.

Yes. I think `context` makes more sense from a user's point of view.